index.html

```diff
@@ -0,0 +1,256 @@
+            <li>
+              <a href=""><i class=" nav bi-facebook"></i></a>
+          </li>
+          <li>
+              <a href=""><i class="nav bi bi-twitter"></i></a>
+          </li>
+              
+              <li><a href="#" class="nav">English</a></li>
+              <li><a href="#" class="nav">MyPage</a></li>
+              <li><a href="#" class="nav">Layout</a></li>
+          </ul>
+        </div>
+   </header>
+
+   <header class="header2 bg-light">
+    <div class="logo1"><a href="index.html" class="logo1"><img src="./images/churchlogo.jpg" alt="Church Icon" class="churchlogo"></a></div>
+    <nav class="menu-list1">
+        <ul onclick = "hideMenu()">
+            <li><a href="index.html" class="nav">Home</a></li>
+            <li><a href= "about.html" class="nav">About</a></li>
+            <li><a href="#speakers" class="nav">Speakers</a></li>
+            <li><a href="#partners" class="nav">Partners</a></li>
+            <li class="campaign"><a href="#" class="nav border text-center">WC Campaign</a></li>
+
+        </ul>
+    </nav>
+    <div class="navigation"  onclick = "showMenu()">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+    </div>
+ 
+</header>
+   
+     
+
+    <section class="header pt-0 p-3" id="header">
+        <div class="container-xxl" id="bg-parent">
+           <div class="row">
+            <div class="position-absolute shadow-lg " id="bg-img">
+                <div class=" align-items-end img-heading" id="img-heading">
+                    <h2 class="display-5 text-warning">
+                        "Revival in the City!"
+                    </h2><br><br>
+                    <h1 class="thestory">
+                      The Glory of the Lord <br>reigns here today! <br>2022
+                  </h1> 
+                </div>
+                  <div class="p-3 border text-center bg-white col-md-5 text-center" id="main-p">
+                    <p class>Lorem Ipsum is simply dummy text of the printing and
+                       typesetting industry. Lorem Ipsum has been the industry's standard
+                        dummy text ever since the 1500s, when an unknown printer took a galley of
+                         type and scrambled it to make a type specimen book. </p>
+                         
+                          </div>
+                <div class="p-3 text-center bg-white col-md-5" id="main-h">
+                  <h2 class="display-5 fw-bold lead text-secondary" id="revival-date">
+                    2022.07.04.(MON) ~ 10(SUN)
+                  </h2>
+                  <h6 class="display-4 font-weight-lighter lead text-secondary" id="revivaldate">
+                    @Wells First Community Church.
+                  </h6>
+                  
+              </div>
+            </div>
+           </div>
+        </div>        
+   
+    </section>
+
+
+    
+<section class="m-auto pt-5" id="program">
```

Comment:
In this section, individual articles should have their elements organized in a vertical column, with the image at the top, title in the center, and description at the bottom, and follow the original design.
your design:
![mainp](https://user-images.githubusercontent.com/53356820/176939802-e3b90d6e-dec7-487f-be4d-328794b61a5a.png)
Remove the red border-color